jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerFactory.kt

```diff
@@ -30,6 +30,8 @@ class AwsExplorerFactory : ToolWindowFactory, DumbAware {
             toolWindow.setTitleActions(
                 actionManager.getAction("aws.settings.refresh"),
                 Separator.create(),
+                actionManager.getAction("aws.toolkit.startTerminal"),
```

Comment:
what happens if this is null (due to plugin not enabled)?

Reply:
that ... is a great question - will test.